### PR TITLE
Added...uhh, what was it? Oh right, Dementia

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Adds Dementia as a trait (Credit to Hunter on Discord for the idea and list of words)

## Why / Balance
Good extra bit of fun for people playing older characters (Also fun for admins to bully metagamers)

## Technical details
Added dementia as an accent

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

:cl:
- add: Added...uhh, what was it? Oh right, Dementia

## Summary by CodeRabbit

- **New Features**
	- Introduced a new "Dementia" accent that dynamically alters speech messages with unique prefixes and suffixes.
	- Added a new trait for characters experiencing dementia, enhancing character interactions.
	- Implemented a localization file for the dementia accent, providing contextually relevant phrases and word replacements.

- **Bug Fixes**
	- No bug fixes were included in this release.

- **Documentation**
	- Updated localization and trait descriptions to include the new dementia accent features.

### CODE REVIEW

**Comment by deltanedas** [comment]
sleepy joe rp

**Comment by a-person5660** [comment]
make it randomly insert stuff like "uuuuh... oh right," and stuff like that. would be very cool.

**Comment by HenhenIII** [comment]
you just made the only 4 players that play an older character happy. I approve this

**Comment by deltanedas** [comment]
uhhh... i forgot, anyway

**Comment by TurboTrackerss14** [comment]
These FTL stinkgs are going to break normal conversation. 

"Daniel smithson is a ling" becomes "Danisl smithson is a some sort of bad guy, forgot which"

File: Resources/Locale/en-US/accent/dementia.ftl [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+accent-dementia-words-22 = heretic
+accent-dementia-words-replace-22 = some sort of bad guy, forgot which
+accent-dementia-words-23 = heretics
+accent-dementia-words-replace-23 = some sort of bad guys, forgot which
+accent-dementia-words-24 = ninja
+accent-dementia-words-replace-24 = some sort of bad guy, forgot which
+accent-dementia-words-25 = deathsquad
+accent-dementia-words-replace-25 = some sort of bad guy, forgot which
+accent-dementia-words-26 = thief
+accent-dementia-words-replace-26 = some sort of bad guy, forgot which
```
[the remark this answers is not in the document]
Comment (TurboTrackerss14) [reply Resources/Locale/en-US/accent/dementia.ftl:63]: Additionally should have both abbreviations AND full names
Revs + revolutionary 
headrev + head revolutionary 
Syndie + syndicate 
Syndie + traitor 
ling + changeling

**Comment by BeeRobynn** [comment]
> These FTL stings are going to break normal conversation.
> 
> "Daniel smithson is a ling" becomes "Danisl smithson is a some sort of bad guy, forgot which"

Dementia breaks normal conversation

**Comment by Aidenkrz** [comment]
Leaving this open as it feels kinda lackluster right now, I recommend looking at ohio accent for common words that should be modified.

**Pull request merged**